src/securityPolicy.c

```diff
@@ -282,9 +282,18 @@ security_policy_t policyForSignTxOutputAssetGroup(
         token_group_t* tokenGroup MARK_UNUSED
```

Comment:
is there any point in having this function which just mirrors the `addressPolicy`? Why not directly using `addressPolicy` within the function to handle asset groups/tokens to make it clear they should share the same policy?

Reply:
I would perhaps rename the `subctx->addressSecurityPolicy` to `subctx->outputSecurityPolicy` to reflect that if you agree with my suggestion

Reply:
and same comment for `policyForSignTxOutputToken`